src/node.rs

```diff
@@ -10,32 +12,28 @@ use crate::error::{TaffyError, TaffyResult};
 use crate::geometry::Size;
 use crate::layout::Layout;
 use crate::style::FlexboxLayout;
-#[cfg(any(feature = "std", feature = "alloc"))]
-use crate::sys::Box;
 use crate::sys::{new_vec_with_capacity, ChildrenVec, Vec};
 use crate::{data::NodeData, error};
 
-/// A function type that can be used in a [`MeasureFunc`]
-///
-/// This trait is automatically implemented for all types (including closures) that define a function with the appropriate type signature.
-pub trait Measurable: Send + Sync + Fn(Size<Option<f32>>) -> Size<f32> {}
-impl<F: Send + Sync + Fn(Size<Option<f32>>) -> Size<f32>> Measurable for F {}
-
-/// A function that can be used to compute the intrinsic size of a node
-pub enum MeasureFunc {
-    /// Stores an unboxed function
-    Raw(fn(Size<Option<f32>>) -> Size<f32>),
+/// Trait that can measure nodes using previously stored associated data
+pub trait Measure<T> {
+    fn measure(&self, node: Node, data: &T, constraint: Size<Option<f32>>) -> Size<f32>;
+}
 
-    /// Stores a boxed function
-    #[cfg(any(feature = "std", feature = "alloc"))]
-    Boxed(Box<dyn Measurable>),
+pub struct NoMeasure {
+    phantom: PhantomData<()>,
 }
 
+pub type SimpleTaffy = Taffy<NoMeasure>;
```

Comment:
Default type args will probably do a better job of this :)

Reply:
I tried but the code in tests was not happy with `Taffy::new()` (when no measurements needed), it kept asking me for explicit annotations. But maybe I misunderstood how to properly provide default type args